src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -632,6 +632,11 @@ Public Class BuildDevDivInsertionFiles
                         Continue For
                     End If
 
+                    If partFileName.EndsWith(".resources.dll") Then
```

Comment:
@tmat Don't include the .resource.dll files in the main VSIX--that will cause us to install all of them regardless of the lang packs the user selects in the VS Installer.

I'm in the process of recreating the resource-specific VSIXes--which used to build in the VS repo--in the dotnet/roslyn repo. I'll handle laying out the resource DLLs for InteractiveHost*.exe in the new locations as part of that.

Reply:
Do you mean that the resources will be included in a different VSIX than the binaries and VS assembly loader is able to find them there?
That won't work for Interactive Host though, since Interactive Host process has no knowledge of Visual Studio.

Reply:
They will be in a different VSIX, yes, but a VSIX that installs to the same directory as the Language Service VSIX. That's already how our resource assemblies are handled in VS--I'm just moving the generation of those VSIXes out of the VS repo and into the dotnet/roslyn repo.

Reply:
Oh, I see. OK.